mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java

```diff
@@ -32,6 +40,7 @@
 public class SyncSurveysDialogFragment extends DialogFragment implements SyncSurveysDialogMvpView {
 
     public static final String LOG_TAG = SyncSurveysDialogFragment.class.getSimpleName();
+    public static final String CHANNEL_ID = "1001";
```

Comment:
Avoid using Hard Coded strings in the code. Create a value in strings.xml and use it from there

Reply:
Ok. Will take care of it. Thanks for telling me.

Reply:
@Grandolf49 I have made the required changes.